### ISSUE TITLE
Incompatible with ajv v8.6.3

### ISSUE THREAD

**Issue by tcyrus** [jdthorpe/ajv-semver#8]
```
these parameters are deprecated, see docs for addKeyword
~\something\node_modules\ajv\dist\compile\util.js:174
        throw new Error(msg);
              ^

Error: strict mode: unknown keyword: "bool_or_ref"
    at checkStrictMode (~\something\node_modules\ajv\dist\compile\util.js:174:15)
    at checkUnknownRules (~\something\node_modules\ajv\dist\compile\util.js:32:13)
    at checkKeywords (~\something\node_modules\ajv\dist\compile\validate\index.js:120:34)
    at validateFunctionCode (~\something\node_modules\ajv\dist\compile\validate\index.js:19:9)
    at Ajv.compileSchema (~\something\node_modules\ajv\dist\compile\index.js:81:45)
    at Ajv._compileMetaSchema (~\something\node_modules\ajv\dist\core.js:476:37)
    at Ajv._compileSchemaEnv (~\something\node_modules\ajv\dist\core.js:464:18)
    at Ajv.compile (~\something\node_modules\ajv\dist\core.js:153:38)
    at Ajv.keywordMetaschema (~\something\node_modules\ajv\dist\core.js:603:31)
    at Ajv.addKeyword (~\something\node_modules\ajv\dist\core.js:350:27)
```

**Comment by jdthorpe** [comment]
Which parameters are deprecated?  A stack trace without knowing anything about how you were using the library when the error was raised makes it impossible to debug.  Please provide a minimal reproducible example that raises this error.

**Comment by jdthorpe** [comment]
Fixed in v1.0.5 (Thanks tcyrus!)
